packages/admin/cms-admin/src/contentScope/Provider.tsx

```diff
@@ -127,6 +128,7 @@ export function useContentScope<S extends ContentScopeInterface = ContentScopeIn
         setRedirectPathAfterChange: context.setRedirectPathAfterChange,
         supported: Object.keys(scope).length > 0,
         values: context.values as ContentScopeValues<S>, // @TODO:
+        createUrl: context.location.createUrl,
```

Comment:
I don't think it's a good idea to create the url myself, so I added the createUrl function so I can use it in the WarningsGrid.
Unless there is already another way to do that (but I think not, because usually it's not needed to switch to another scope I guess)

Reply:
how does the dependnecies feature create url for a different scope?

Reply:
I just checked it and that does not work:

https://github.com/user-attachments/assets/0a2f4189-9762-4325-b51b-13255756af39

Reply:
@thomasdax98 once tried to fix that, but we never finished it: https://github.com/vivid-planet/comet/pull/2493